sidebars.ts

```diff
@@ -23,13 +23,13 @@ module.exports = {
       link: {type: 'doc', id: 'get-started/index'},
       items: [
         'get-started/overview',
+        'get-started/sumo-logic-ui',
```

Comment:
I think you should place `get-started/sumo-logic-ui-classic` here too. I think it would be a good idea to have the new and classic articles listed one after the other in the left nav until the classic UI is removed.

Reply:
I was going to ask you about that - thank you - done!